src/api/spec/features/webui/users/users_ui_spec.rb

```diff
@@ -0,0 +1,62 @@
+require "browser_helper"
+
+RSpec.feature "User's UI shows", type: :feature, js: true do
```

Comment:
Oh shit I was wrong! It was [`:vcr`](https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/spec_helper.rb#L65) that is default not `:js`.  Sorry for the noise